docs/tutorials/flight_controller_telemetry_tutorial.rst

```diff
@@ -0,0 +1,395 @@
+    `PX4 documentation <https://docs.px4.io/main/en/peripherals/dshot.html#telemetry>`_, you may see an error warning that there was a 
+    "Packet length mismatch" as shown below. This does not mean that your telemetry is improperly configured. This error occurs because as of PX4 v1.13.3 the 
+    structure of the DSHOT Info message sent by Vertiq modules does not match the structure expected by PX4. If you see this error,
+    it does not indicate a problem with your setup, move onto the instructions below to confirm that your telemetry is working.
+
+    .. figure:: ../_static/tutorial_images/fc_telemetry_tutorial/dshot_info_error_px4.png
+        :align: center
+        :width: 50%
+
+Testing Telemetry
+##################
+To test that telemetry is working, you can configure your `instrument panel <https://docs.qgroundcontrol.com/master/en/FlyView/FlyView.html#instrument_panel>`_ 
+in QGroundControl to display live ESC Status values. For the example shown below, the instrument panel was configured to show Rpm1, Voltage1, and Current1
+for ESCStatus, in order to show the RPM, Voltage, and Current values from the telemetry received from the module the flight controller considers Motor 1.
+If your module is powered and connected to the flight controller properly, live values for these data points should be shown in the instrument panel as shown below.
+
+.. figure:: ../_static/tutorial_images/fc_telemetry_tutorial/px4_dshot_instrument_panel.png
+    :align: center
+    :width: 40%
+    :alt: QGroundControl Instrument Panel Showing DSHOT Telemetry
+
+    QGroundControl Instrument Panel Showing DSHOT Telemetry
+
+To see what the telemetry would look like when spinning, you can use the "motor_test" command in the MAVLink Console under Analyze Tools, as shown
+in the code block below.
+
+.. code::
+
+    motor_test test -m 1 -p 5
+    INFO  [motor_test] motor 1 set to 0.05
+    nsh> 
+
+The image below shows an example of what the instrument panel may look like when the module is spinning. 
+
+.. figure:: ../_static/tutorial_images/fc_telemetry_tutorial/px4_dshot_instrument_panel_spinning.png
+    :align: center
+    :width: 40%
+    :alt: QGroundControl Instrument Panel Showing DSHOT Telemetry When the Module is Spinning
+
+    QGroundControl Instrument Panel Showing DSHOT Telemetry When the Module is Spinning
+
+Note that if you are using the default settings for MOT_POLE_COUNT, the RPM shown on the instrument panel will be significantly higher than the actual RPM of the module. This is because PX4 expects the module
+to report ERPM, but Vertiq modules report RPM directly as covered in the :ref:`ESC Telemetry <manual_telemetry>` section of the Feature Reference Manual.
+To learn how to adjust the MOT_POLE_COUNT to properly show RPM, see the :ref:`fc_telemetry_erpm_to_rpm` section below.
+
+.. _fc_telemetry_erpm_to_rpm:
+
+Converting ERPM to RPM Workaround

```

Comment:
Same story as with ArduCopter, PX4 expects ERPM. But we can do a hacky workaround by setting the motor poles to 200. Non-ideal, but until we do something to improve integration on either end of this, its the best way for people to get accurate RPM data